projectroles/utils.py

```diff
@@ -249,7 +249,7 @@ def get_project_app_links(
                             'projectroles:update',
                             kwargs={'project': project.sodar_uuid},
                         ),
-                        'label': 'Update Project',
+                        'label': f'Update {get_display_name(project.type)}',
                         'icon': 'mdi:lead-pencil',
                         'active': self._is_active_projectroles(
                             link_names=['update'],
```

Comment:
For all the links after this point in the code, all the labels are still provided as hardcoded text. Please go through all the changes required carefully before submitting.